### PR TITLE
Monitoring resources

### PR DESCRIPTION
**Pull request by pb82**
Update monitoring to v1.0.0

Verification steps:

* Install monitoring using this branch
* Make sure that all operators, prometheus, alertmanager and grafana come up successfully
* Test with other products and make sure that all alerts, dashboards and blackboxtargets are imported

### CODE REVIEW

File: manifests/integreatly-monitoring/monitoring-1.0.0/application-monitoring-operator.v1.0.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+        kind: GrafanaDashboard
+        displayName: Integreatly monitoring Grafana dashboard.
+        description: Represents an instance of the Integreatly Grafana dashboard.
+      - name: grafanadatasources.integreatly.org
+        version: v1alpha1
+        kind: GrafanaDataSource
+        displayName: Integreatly monitoring Grafana datasource.
+        description: Represents an instance of the Integreatly Grafana datasource.
+  description: Placeholder description
+  displayName: Application Monitoring Operator
```
Comment (davidkirwan) [inline manifests/integreatly-monitoring/monitoring-1.0.0/application-monitoring-operator.v1.0.0.clusterserviceversion.yaml]: Might need to add `replaces: application-monitoring-operator.v0.0.28`
like: https://github.com/integr8ly/manifests/blob/master/integreatly-monitoring/monitoring-0.0.28/application-monitoring-operator.v0.0.28.clusterserviceversion.yaml#L45

Comment (pb82) [reply manifests/integreatly-monitoring/monitoring-1.0.0/application-monitoring-operator.v1.0.0.clusterserviceversion.yaml]: hmm, good point. But will we ever upgrade from 0.0.28?

Comment (davidkirwan) [reply manifests/integreatly-monitoring/monitoring-1.0.0/application-monitoring-operator.v1.0.0.clusterserviceversion.yaml]: Hmm I guess not! Just looked all the manifests in quay have been yanked.

**Comment by StevenTobin** [comment]
You pulled in my alert test commit, if you add this commit it'll get rid of the 3scale alerts we don't need anymore https://github.com/integr8ly/integreatly-operator/pull/255/commits/40a176f4260fa8445fb7ba35c42e59b30b5267b2

**Pull request merged**